scale/storage/models.py

```diff
@@ -217,7 +217,23 @@ def download_files(self, file_downloads):
             wp_file_downloads = wp_dict[wp_id][1]
             workspace.download_files(wp_file_downloads)
 
-    def filter_files(self, started=None, ended=None, time_field=None, file_name=None):
+    def get_details(self, file_id):
+        """Returns the file for the given ID with all detail fields included.
+
+        There are currently no additional fields included.
+
+        :param file_id: The unique identifier of the workspace.
+        :type file_id: int
+        :returns: The file with all detail fields included.
+        :rtype: :class:`storage.models.ScaleFile`
+        """
+
+        # Attempt to get the workspace
+        file = ScaleFile.objects.get(pk=file_id)
```

Comment:
Use `scale_file` instead of `file` since I think `file` is a builtin object/function.